catalog/transformer/catalog-transformer-mgmp/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -61,4 +62,13 @@
             <reference interface="ddf.catalog.data.AttributeRegistry"/>
         </argument>
     </bean>
+
+    <service ref="mgmpTransformer" interface="ddf.catalog.transform.MetacardTransformer" ranking="10">
+        <service-properties>
+            <entry key="id" value="gmd:MD_Metadata"/>
```

Comment:
Please rename gmd:MD_Metadata to MGMPv2 to avoid duplicate listings in the Action tab of the UI